### PR TITLE
Add 19/10/11 Bangalore event

### PR DESCRIPTION
**Pull request by saskali**
A bit late, but here we go. 

Adding the Bangalore post for https://github.com/ClojureBridge/Workshops/issues/98. 

Tried running it locally which seems fine, feedback welcome!

### CODE REVIEW

File: _posts/2019-10-11-bangalore.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout:             post
+permalink:          /events/2019-10-11-bangalore
+country:            India
+city:               Bangalore
+street:             Indiranagar Stage 1
+latitude:           12.981880
+longitude:          77.639005
```
Comment (qmg-awhitehouse) [inline _posts/2019-10-11-bangalore.md]: I checked the original location given on Nivedita's instructions and it's slightly different 12.981890, 77.638548

Comment (saskali) [reply _posts/2019-10-11-bangalore.md]: ahh good catch, thanks! I'll adjust it now

File: _posts/2019-10-11-bangalore.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+city:               Bangalore
+street:             Indiranagar Stage 1
+latitude:           12.981890
+longitude:          77.638548
+start-date:         2019-10-11
+end-date:           2019-10-12
+registration-url:   https://www.meetup.com/meetup-group-JrOdKOTm/events/264891750
+city-image-url:     assets/images/events/bangalore.jpg
+gravatar-email:     nid@nilenso.com
+organizers:         [{"email": "nid@nilenso.com", "github": "nid90", "name": "Nivedita Priyadarshini", "twitter": "nid90"}, {"email": "kitallis@nilenso.com", "github": kitallis, "name": "Akshay Gupta", "twitter": kitallis}]
```
Comment (davidchambers) [inline _posts/2019-10-11-bangalore.md]: I don't know YAML very well. Do strings need to be quoted? Regardless, perhaps we should use quotes to be consistent:

```suggestion
organizers:         [{"email": "nid@nilenso.com", "github": "nid90", "name": "Nivedita Priyadarshini", "twitter": "nid90"}, {"email": "kitallis@nilenso.com", "github": "kitallis", "name": "Akshay Gupta", "twitter": "kitallis"}]
```

Comment (saskali) [reply _posts/2019-10-11-bangalore.md]: good point, I'll put them in quote. thanks for reviewing David!

Comment (davidchambers) [reply _posts/2019-10-11-bangalore.md]: No problem at all. :)

**Pull request merged**